addon/components/login-modal.js

```diff
@@ -2,18 +2,26 @@ import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
+import config from "ember-ebau-gwr/config/environment";
```

Comment:
As discussed: For the sake of clarity we should use a different name so it is not confused with the `config` service.

Reply:
Done :+1: